packages/perseus/src/widgets/image/__snapshots__/image.test.ts.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`image widget - isMobile %b should snapshot: first render 1`] = `
```

Comment:
These snapshots got re-ordered because I fixed the boolean format.  I think the `true` version used to be first here, but now `false` is.